src/settings/_colours-applied.scss

```diff
@@ -127,4 +136,4 @@ $govuk-link-hover-colour: govuk-colour("light-blue") !default;
 /// @type Colour
 /// @access public
 
-$govuk-link-active-colour: govuk-colour("light-blue") !default;
+$govuk-link-active-colour: govuk-colour("light-blue") !default;
```

Comment:
Minor, but looks like you've removed the new line from the end of this file.

Reply:
Looks like the linter's picked up on this as well, which I think is why the tests (and the review app) are now failing.

Reply:
All hail the linter